src/cmds/scripts/pbs_habitat.in

```diff
@@ -717,6 +717,24 @@ if [ "${PBS_START_SERVER:-0}" != 0 ] ; then
 		fi
 		${PBS_EXEC}/bin/qmgr <<-EOF > /dev/null
 			create queue workq
+		EOF
+		ret=$?
+		if [ $ret -ne 0 ]; then
+			tries=3
+			while [ $tries -ge 0 ]
+			do
+				sleep 2
+				${PBS_EXEC}/bin/qmgr <<-EOF > /dev/null
```

Comment:
we should move the initial try as well inside the while

Reply:
I was thinking on faster system let the code goes as it is why should we add 2 sec sleep.

Reply:
I think @prakashcv13 is suggesting something like this...

`
let tries=3
while [ $tries -gt 0 ]; do
    ${PBS_EXEC}/bin/qmgr -c 'create queue workq'
    ret=$?
    [ $ret -eq 0 ] && break
    let tries-=1
done
if [ $ret -ne 0 ]; then
    echo "ERROR!"
fi
`

Reply:
Thanks @mike0042 . I have updated the code.